kopf/clients/fetching.py

```diff
@@ -58,4 +58,16 @@ def list_objs(*, resource, namespace=None):
     cls = classes._make_cls(resource=resource)
     namespace = namespace if issubclass(cls, pykube.objects.NamespacedAPIObject) else None
     lst = cls.objects(api, namespace=pykube.all if namespace is None else namespace)
-    return lst.response
+    rsp = lst.response
+
+    items = []
+    resource_version = rsp.get('metadata', {}).get('resourceVersion', None)
+    for item in rsp['items']:
+        # FIXME: fix in pykube to inject the missing item's fields from the list's metainfo.
```

Comment:
I didn't find an issue on `pykube` for that, hasn't it been reported yet?

Reply:
Nope. Not sure it should be there either. It is a way how K8s API works.